### PR TITLE
fix: avoid generated typenames conflicts between <opName> and <opName>Internal

### PR DESCRIPTION
**Pull request by fiam**
Refactor the typenames to a single place, then rename the generated types using
a different naming schema that's less likely to cause clashes.

### CODE REVIEW

File: packages/sdk/src/codegen/templates/typescript/internal.client.template.ts [hunk 1]
```diff
@@ -9,15 +9,15 @@ import { {{ modelImports }} } from "./models"
...
+        {{operationName}}: ({{#if hasInternalInput}}options: OperationArgsWithInput<{{internalInputTypename}}>{{/if}}) => Promise<{{#if isTypeScriptOperation}}{ data?: {{responseDataTypename}}, errors?: OperationErrors['{{operationPath}}'][] }{{else}}{ data?: {{responseTypename}}['data'], errors?: Required<{{responseTypename}}>['errors'] }{{/if}}>;
     {{/each}}
 {{/if}}
 }
 
 export interface Mutations  {
 {{#if hasInternalMutations}}
     {{#each internalMutations}}
-        {{operationName}}: ({{#if hasInternalInput}}options: OperationArgsWithInput<{{operationName}}Input>{{/if}}) => Promise<{{#if isTypeScriptOperation}}{ data?: {{operationName}}ResponseData, errors?: OperationErrors['{{operationPath}}'][] }{{else}}{ data?: {{operationName}}Response['data'], errors?: Required<{{operationName}}Response>['errors'] }{{/if}}>;
```
Comment (fiam) [inline packages/sdk/src/codegen/templates/typescript/internal.client.template.ts:20]: @jensneuse This was using `<operationName>Input` but that seemed incorrect to me and I changed it to use the type for the internal input (i.e. `Internal<operationName>Input`). PTAL

File: packages/sdk/src/codegen/templates/typescript/helpers.ts [hunk 2]
```diff
@@ -63,11 +71,21 @@ export const liveQueries = (application: ResolvedApplication, includeInternal: b
...
+				injectedInputTypename: operationInjectedInputTypename(op),
+				internalInputTypename: operationInternalInputTypename(op),
 				liveQuery: true,
 				requiresAuthentication: op.AuthenticationConfig.required,
+				responseDataTypename: operationResponseDataTypename(op),
+				responseTypename: operationResponseTypename(op),
 			};
 		});
 
+export const operationInputTypename = (op: GraphQLOperation) => `${op.Name}Input`;
```
[the remark this answers is not in the document]
Comment (jensneuse) [reply packages/sdk/src/codegen/templates/typescript/helpers.ts:83]: Have you considered using `#` as part of the name? It makes it extremely unlikely that someone uses it.

File: packages/testsuite/apps/basic/.wundergraph/operations/nested/InternalWrapper.ts [hunk 3]
```diff
@@ -2,6 +2,6 @@ import { createOperation } from '../../generated/wundergraph.factory';
 
 export default createOperation.query({
 	handler: async ({ internalClient }) => {
-		return internalClient.queries.NestedInternal();
+		return internalClient.queries.NestedInternalInternal();
```
Comment (jensneuse) [inline packages/testsuite/apps/basic/.wundergraph/operations/nested/InternalWrapper.ts:5]: This can be refactored to use the operations client, right?

**Review by jensneuse — APPROVED** [review]
great improvements, one suggestions regarding generated names, see comment

File: packages/testsuite/apps/basic/.wundergraph/operations/nested/InternalWrapper.ts (hunk 3 above)
Comment (fiam) [reply packages/testsuite/apps/basic/.wundergraph/operations/nested/InternalWrapper.ts:5]: We could, but I think it's a good idea to keep it around until we delete the internal client.

File: packages/sdk/src/codegen/templates/typescript/helpers.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (fiam) [reply packages/sdk/src/codegen/templates/typescript/helpers.ts:83]: I don't think `#` would be valid, but you've made me realize `$` would be a very good choice. I'll update it.

**Review by jensneuse — APPROVED** [review]

**Pull request merged**